run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -2413,7 +2413,7 @@ Warnings:                    1
 )))HTAP_SHIP
 
 (((CEDSv2_SHIP
-102 CEDS_NO_SHP $ROOT/CEDS/v2021-06/$YYYY/NO-em-anthro_CMIP_CEDS_$YYYY.nc NO_shp 1750-2010/1-12/1/0 C xy kg/m2/s NO 25 10 5
+102 CEDS_NO_SHP $ROOT/CEDS/v2021-06/$YYYY/NO-em-anthro_CMIP_CEDS_$YYYY.nc NO_shp 1750-2019/1-12/1/0 C xy kg/m2/s NO 25 10 5
```

Comment:
Question: does the CEDS_NO_SHP only go up to 2019?  The other emissions go to 2020.  Just curious.

Reply:
I see that it does. Good to go.